l10n_eu_oss/wizard/l10n_eu_oss_wizard.py

```diff
@@ -0,0 +1,214 @@
+# Copyright 2021 Valentin Vinagre <valentin.vinagre@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, fields, models
+from odoo.exceptions import Warning
+
+
+class L10nEuOssWizard(models.TransientModel):
+    _name = "l10n.eu.oss.wizard"
+    _description = "l10n.eu.oss.wizard"
+
+    def _get_default_company_id(self):
+        return self.env.company.id
+
+    def _get_eu_res_country_group(self):
+        eu_group = self.env.ref("base.europe", raise_if_not_found=False)
+        if not eu_group:
+            raise Warning(
+                _(
+                    "The Europe country group cannot be found. "
+                    "Please update the base module."
+                )
+            )
+        return eu_group
+
+    def _default_fiscal_position_id(self):
+        user = self.env.user
+        eu_id = self._get_eu_res_country_group()
+        return self.env["account.fiscal.position"].search(
+            [
+                ("company_id", "=", user.company_id.id),
+                ("vat_required", "=", True),
+                ("country_group_id.id", "=", eu_id.id),
+            ],
+            limit=1,
```

Comment:
Could it be better to order by `sequence` to get the first fiscal position if more than one?

Reply:
The model is ordered by sequence https://github.com/odoo/odoo/blob/fa2fe075055c50567f204d82d7db7a8744a15762/addons/account/models/partner.py#L18

Reply:
:+1: